cmake/modules/hunter_autotools_configure_options.cmake

```diff
@@ -0,0 +1,176 @@
+# Copyright (c) 2015 Ruslan Baratov, Alexandre Pretyman
```

Comment:
Same here. I think this is quite big change so it can be separate pull request for easy review reason.